### DISCUSSION TITLE
SSH authentication problem from aws-ubuntuec2 instance.

### DISCUSSION THREAD

**Discussion by Sujit-Baram** [community/community#40413] [New to GitHub]
HI every one i am trying clone my repo to my aws ubuntu-ec2 instance . But its getting error 'Permission Denied (publickey)' 

I am adding my ssh public key in git ssh key authentication. But still error showing,

For ssh key generation i am user the command $ ssh-keygen        from my aws Ubuntu-ec2 console

**Comment by ossanna16** [comment]
Hi there @Sujit-Baram and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
